services/dedup/setup.go

```diff
@@ -23,3 +28,12 @@ func GetInstance(clearDB *bool) DedupI {
 
 	return dedupManager
 }
+
+func cleanUpV2Data() {
+	// clean up badgerDBv2 data
+	tmpDirPath, err := misc.CreateTMPDIR()
+	if err != nil {
+		return
+	}
+	_ = os.RemoveAll(fmt.Sprintf(`%v/badgerdbv2`, tmpDirPath))
```

Comment:
before deleting these, shouldn't we migrate them first to the new db?

Reply:
I checked to official documentation and read somewhere that since the way data is stored changed v2 to v3 carrying out the migration would be very tough. Since dedup was not being used my many customers back then I went ahead with this but now this would require a bit more rethinking since we are planning to enable it for everyone from this release